### PR TITLE
feat(home): Added work item component to home page

### PR DESCRIPTION
**Pull request by dlabrecq**
This work item component displays work items assigned to the user.
The select menu allows the user to select from the latest 10 spaces.
By default, the component will select a recent space which is populated with work items.
Reviewed with Catherine.

screen shot 2017-04-20 at 11 48 52 am

### CODE REVIEW

**Comment by fabric8cd** [comment]
@dlabrecq snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-932-fabric8-ui.int.rdu2c.fabric8.io

File: src/app/getting-started/getting-started.component.html [hunk 1]
```diff
@@ -44,41 +44,41 @@ <h2 class="circle">2</h2>
             Please connect your GitHub and OpenShift accounts to OpenShit.io.
           </p>
           <div *ngIf="gitHubLinked && openShiftLinked; then allLinked else allUnlinked"></div>
-          <template #allLinked>
+          <ng-template #allLinked>
```
Comment (joshuawilson) [inline src/app/getting-started/getting-started.component.html]: This page looks like it is not related to the PR or commit.

**Comment by joshuawilson** [comment]
@dlabrecq this needs to be rebased it is missing changes from master that are on the home dashboard.

File: src/app/getting-started/getting-started.component.html (hunk 1 above)
Comment (dlabrecq) [inline src/app/getting-started/getting-started.component.html]: While coding this page, I remembered we updated all templates to ng-template for Angular 4. This one slipped through with the getting started page. I can create a separate PR if you prefer?

**Comment by fabric8cd** [comment]
@dlabrecq snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-932-fabric8-ui.int.rdu2c.fabric8.io

File: src/app/home/home.component.html [hunk 2]
```diff
@@ -134,7 +97,7 @@ <h2 class="card-pf-title">
             <!-- A list of the 5 most recent Spaces will be displayed in place of the Blank Slate. Additional Spaces are viewed by using the 'See All' button in the Card footer -->
             <div class="home-space-list-result" *ngIf="_spaces.length > 0">
               <ul class="list-group list-view-pf list-view-pf-view">
-                <li class="list-group-item" *ngFor="let space of recent">
+                <li class="list-group-item" *ngFor="let space of _spaces">
```
Comment (joshuawilson) [inline src/app/home/home.component.html]: This should not be changed. Did you have a reason to change this?

File: src/app/home/home.component.html [hunk 3]
```diff
@@ -74,51 +75,13 @@ <h2 class="card-pf-title text-center">
...
             <div class="card-pf-heading-details card-f8-heading-details" *ngIf="_spaces.length > 0">
               <a class="btn btn-link" (click)="createSpace.open()">
                 <span>Create a space</span>
                 <i class="pficon pficon-add-circle-o"></i>
               </a>
             </div>
-            <h2 class="card-pf-title">
-              <a href="#">Spaces</a>
-            </h2>
+            <h2 class="card-pf-title">My spaces</h2>
```
Comment (joshuawilson) [inline src/app/home/home.component.html]: You might even add the word "recent", maybe

File: src/app/home/home.component.html (hunk 2 above)
Comment (dlabrecq) [inline src/app/home/home.component.html]: This may be a bad merge. I'll start over.

**Pull request closed without merging**